src/_tkagg.cpp

```diff
@@ -256,25 +256,32 @@ int load_tcl(T lib)
 
 void load_tkinter_funcs(void)
 {
-    // Load Tcl/Tk functions by searching all modules in current process.
-    HMODULE hMods[1024];
-    HANDLE hProcess;
-    DWORD cbNeeded;
-    unsigned int i;
+    HANDLE process = GetCurrentProcess();  // Pseudo-handle, doesn't need closing.
+    HMODULE* modules = NULL;
+    DWORD size;
+    if (!EnumProcessModules(process, NULL, 0, &size)) {
+        PyErr_SetFromWindowsErr(0);
+        goto exit;
+    }
+    if (!(modules = reinterpret_cast<HMODULE*>(malloc(size)))) {
```

Comment:
It's more of a `static_cast`, isn't it? `void*` has no form and thus can't be (re)interpreted.

Reply:
Ah, yes.